src/blueprints/Do.Blueprints.Service.Application/Authentication/FixedBearerToken/AddHashParameterToFormPostConvention.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Do.Domain.Model;
+using Do.RestApi.Configuration;
+using Newtonsoft.Json;
+using System.ComponentModel.DataAnnotations;
+
+namespace Do.Authentication.FixedBearerToken;
+
+public class AddHashParameterToFormPostConvention(DomainModel _domainModel) : IApiModelConvention<ActionModelContext>
```

Comment:
we've been using `_domain` instead of `_domainModel` in other conventions. check if they're all `_domain` if so, change this to `_domain`, if not leave it